docs/binaries/santabundleservice.md

```diff
@@ -0,0 +1,44 @@
+---
+parent: Binaries
+---
+
+# santabundleservice
+
+The `santabundleservice` is a small launch daemon responsible for creating
+non-execution events for the contents of a bundle. When an execution is blocked,
+the `santabundleservice` is tasked with determining if the binary is part of a
+bundle and, if so, locating other executables contained within that bundle.
+Finally, the service is responsible for generating the
+[bundle hash](#bundle-hash) and computing the hash of all found binaries.
+
+## Bundle Identification
+
+Bundle structures for macOS applications is a loosely defined structure and is
+further complicated by supporting nested bundles for common scenarios (e.g. XPC
```

Comment:
This reads a bit oddly. How about this?

"macOS application bundles are formed of a file folder with a loosely-defined structure"

Reply:
Probably not a bad idea to also link to https://developer.apple.com/library/archive/documentation/CoreFoundation/Conceptual/CFBundles/BundleTypes/BundleTypes.html

Reply:
Fixed, added link